src/amo/components/Errors/UnavailableForLegalReasons/index.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+import * as React from 'react';
+import { compose } from 'redux';
+
+import ErrorComponent from 'amo/components/Errors/ErrorComponent';
+import translate from 'core/i18n/translate';
+import type { I18nType } from 'core/types/i18n';
+
+type Props = {||};
+
+type InternalProps = {|
+  ...Props,
+  i18n: I18nType,
+|};
+
+export class UnavailableForLegalReasonsBase extends React.Component<InternalProps> {
+  render() {
+    const { i18n } = this.props;
+
+    return (
+      <ErrorComponent
+        code={451}
+        header={i18n.gettext('Not Available In Your Region')}
+      >
+        <p>
+          {i18n.gettext(
+            'Sorry, but the content you are seeking is not available in your region.',
```

Comment:
I think we should ask for copy approval from Product on the text and title in the issue.  (I think it's fine as is, but there are some possible variations)

Reply:
I agree. I just put in a guess and fully expect product to tell us what it should actually say. I'll ping Jorge in the issue.